src/frontend/e2e/app.po.ts

```diff
@@ -1,11 +1,78 @@
 import { browser, by, element } from 'protractor';
 
 export class FogLAMPPage {
-  navigateTo() {
+  navigateToHome() {
     return browser.get('/');
   }
 
-  getParagraphText() {
-    return element(by.css('app-root h1')).getText();
+  navigateToConfig() {
+    return browser.get('/configuration');
+  }
+
+  navToScheduledTasks() {
+    return browser.get('/scheduled-tasks');
+  }
+
+  navToAuditLogs() {
+    return browser.get('/audit');
+  }
+
+  navToAssetReadings() {
+    return browser.get('/assets');
+  }
+
+  getNavTitle() {
+    browser.ignoreSynchronization = true;
+    return element(by.css('.container .nav-item b')).getText();
+  }
+
+  getAppStatus() {
+    browser.ignoreSynchronization = true;
+    return element(by.css('#app .nav-right span')).getText();
+  }
+
+  getConfigTitles() {
+    browser.ignoreSynchronization = true;
+    return element(by.css('#app app-configuration-manager')).getText();
+  }
+
+  getTitleSchedules() {
+    browser.ignoreSynchronization = true;
+    return element(by.css('app-scheduled-process .is-parent > div > header p')).getText();
+  }
+
+  getTitleCreateSchedule() {
```

Comment:
getCreateScheduleTitle?

Reply:
with both of above, either we say `getTitle<Component>` or `get<Component>Title` i.e. stay consistent